2 table.sql

```diff
@@ -0,0 +1,31 @@
+use ArmyRecordsDB
+go 
+drop table if exists dbo.Soldiers
+go 
+create table dbo.Soldiers(
+    SoldiersID int not null identity primary key,
+    SoldierLastName varchar(25) not null
+        constraint c_Soldiers_soldier_last_name_cannot_be_blank check(SoldierLastName <> ''),
+    SoldierFirstName varchar(25) not null
+        constraint c_Soldiers_soldier_first_name_cannot_be_blank check(SoldierFirstName <> ''),
+    SoldierSSN char(10) not null
```

Comment:
SSN must be in specific format. It should be 8 digits followed by a hyphen followed by one digit.

Reply:
done. I deleted the not blank constraint also.